ui/src/DataService.js

```diff
@@ -8,7 +8,7 @@ import config from './config/config';
  * @param {object|Array} body - JSON Payload to send in the request
  * @returns a config object to send in HTTP requests
  */
-const requestConfig = (method, body) => ({
+export const requestConfig = (method, body) => ({
```

Comment:
Can this be exported [via the `dataService` object](https://github.com/aerogear/mobile-security-service/blob/2062561c1fdab518763fa2dc91d7c07cdcc2f85b/ui/src/DataService.js#L90-L94) so we are consistent

Reply:
It could, but the reason I'm exporting it separately is because the AuthService does not want the entire DataService. If I export it here then the AuthService would have to import the DataService just to use the `requestConfig`.